src/app/(fullscreen)/trait/result/page.tsx

```
@@ -52,20 +56,41 @@ export default function TraitResult() {
 
   return (
     <div className="relative grid min-h-screen w-full grid-rows-[auto_1fr_auto] overflow-hidden bg-black">
+      {isFromMyPage && (
+        <div className="absolute top-5 left-5 z-50">
+          <button onClick={() => router.back()} className="text-white">
+            ←
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)